### PR TITLE
[WIP] Feature/better type support

### PR DESCRIPTION
**Pull request by roll**
See https://github.com/frictionlessdata/jsontableschema-sql-py/issues/9
Includes https://github.com/frictionlessdata/jsontableschema-sql-py/pull/22

### CODE REVIEW

**Comment by roll** [comment]
@akariv 
please review

File: jsontableschema_sql/storage.py [hunk 1]
```diff
@@ -14,7 +17,7 @@
 
 # Module API
 
-class Storage(object):
+class Storage(Storage):
```
Comment (akariv) [inline jsontableschema_sql/storage.py]: perhaps we import Storage as BaseStorage or something so we don't reuse the exact name, causing confusion and ambiguity?

**Comment by akariv** [comment]
Aside from one comment, looks good.

**Comment by roll** [comment]
fixed base class naming

**Pull request merged**